get_pending_accounts.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+from pywell.entry_points import run_from_cli
+import requests
+
+
+DESCRIPTION = 'Get all pending accounts from Mobilize created before today.'
+
+ARG_DEFINITIONS = {
+    'MOBILIZE_API_KEY': 'API key for mobilize.io',
+    'MOBILIZE_API_SECRET': 'API secret for mobilize.io',
+    'MOBILIZE_API_ROOT': 'Root URL for mobilize.io API',
+    'MOBILIZE_DEFAULT_GROUP_ID': 'The first group members request to join',
+    'VERBOSE': 'Give verbose output if set'
+}
+
+REQUIRED_ARGS = [
+    'MOBILIZE_API_KEY', 'MOBILIZE_API_SECRET', 'MOBILIZE_API_ROOT',
+    'MOBILIZE_DEFAULT_GROUP_ID'
+]
+
+
+def get_pending_accounts(args) -> list:
+    today = datetime.now().strftime('%Y-%m-%d')
+    offset = 0
+    count = 0
+    done = False
+    pending_accounts = []
+    while not done:
+        if args.VERBOSE:
+            print('getting users at offset %s' % offset)
+        response = requests.get(
+            '%s%s' % (
+                args.MOBILIZE_API_ROOT,
+                'users?limit=20&offset=%s' % offset
+            ),
+            auth=requests.auth.HTTPBasicAuth(
+                args.MOBILIZE_API_KEY,
+                args.MOBILIZE_API_SECRET
+            )
+        )
+        if args.VERBOSE:
+            print('status: %s' % response.status_code)
+        if response.status_code == 200:
+            users = response.json()
+            count += len(users)
+            pending_accounts += [
+                user for user in users
```

Comment:
The pending_accounts assignment is a bit hard to parse; could you split some of this out?  (If this is idiomatic for python please ignore.)  Moving the for loop and if statement to the outside would make it a lot easier for me to follow.

Reply:
Also - the parsing code *could* have its own unit test (with a mock response as the input), but this is simple enough I'm not sure it's 100% needed.